src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -31,15 +31,19 @@ public class AnalysisService : IDisposable
         /// Defines the list of Script Analyzer rules to include by default if
         /// no settings file is specified.
         /// </summary>
-        private static readonly string[] IncludedRules = {
+        private static readonly string[] IncludedRules = new string[]
+        {
+            "PSUseToExportFieldsInManifest",
+            "PSMisleadingBacktick",
+            "PSAvoidUsingCmdletAliases",
             "PSUseApprovedVerbs",
+            "PSAvoidUsingPlainTextForPassword",
             "PSReservedCmdletChar",
             "PSReservedParams",
             "PSShouldProcess",
             "PSMissingModuleManifestField",
             "PSAvoidDefaultValueSwitchParameter",
-            "PSUseDeclaredVarsMoreThanAssigments",
-            "PSMisleadingBacktick",
+            "PSUseDeclaredVarsMoreThanAssigments"
```

Comment:
I was testing `PSUseDeclaredVarsMoreThanAssigments` so I added it there and forgot to remove it. The rule is still a bit noisy and I would rather remove it.  Thanks for catching it.